offix/src/main/java/org/aerogear/offix/interceptor/ConflictInterceptor.kt

```diff
@@ -58,6 +67,7 @@ class ConflictInterceptor(private val conflictResolutionImpl: ConfliceResolution
 
                 conflictResolutionImpl.resolveConflict(serverStateMap, clientStateMap, conflictedMutationClass)
             } else {
+                userCallback=  queueCallback.removeFirst()
```

Comment:
We will need to use this `userCallback` in `onFailure` as well so maybe make it a global variable inside the `OffixConflictCallback` inner class?